recipes/globus-cli/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "globus-cli" %}
+{% set version = "1.9.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/globus/globus-cli/archive/{{ version }}.tar.gz
+  sha256: e577b1e53e5f2265fa1f08549e26c8bdf6dbc90c22722dd46277db08c13ac38e
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  entry_points:
+    - globus = globus_cli:main
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+    - globus-sdk ==1.6.1
+    - click >=6.6,<7.0
+    - jmespath ==0.9.2
+    - configobj >=5.0.6,<6.0.0
+    - requests >=2.0.0,<3.0.0
+    - six >=1.10.0,<2.0.0
+    - cryptography >=1.8.1,<2.5.0
+
+test:
+  imports:
+    - globus_cli
+    - globus_cli.commands
+    - globus_cli.commands.bookmark
+    - globus_cli.commands.config
+    - globus_cli.commands.endpoint
+    - globus_cli.commands.endpoint.permission
+    - globus_cli.commands.endpoint.role
+    - globus_cli.commands.endpoint.server
+    - globus_cli.commands.task
+    - globus_cli.helpers
+    - globus_cli.parsing
+    - globus_cli.safeio
+    - globus_cli.services
+  commands:
+    - globus --help
+
+about:
+  home: https://github.com/globus/globus-cli
+  license: Apache 2.0
+  license_family: APACHE
+  license_file: LICENSE
+  summary: Globus CLI
+  description: A Command Line Interface to Globus.
+  doc_url: https://globus.github.io/globus-cli
+  dev_url: https://github.com/globusonline/globus-cli
+
+extra:
+  recipe-maintainers:
+    - xylar
+    - lukaszlacinski
```

Comment:
@lukaszlacinski Are you happy to be listed as a maintainer of this recipe?

Reply:
Yes.